operation/fhir-operation-validate/src/main/java/org/linuxforhealth/fhir/operation/validate/ValidateOperation.java

```diff
@@ -131,20 +120,66 @@ private OperationOutcome buildResourceValidOperationOutcome(List<Issue> issues)
     
     
     /**
-     * Check if a resource validation mode code is valid.
+     * This method does the following validations.
+     * 1. Validate if a resource validation mode code is valid.
+     * 2. Validate an interaction for a specified resource type.
+     * 3. Validate if persistence layer implementation supports update/create mode if mode = update/create.
+     * 4. Validate if modes update and delete are only be used when the operation is invoked at the resource instance level
      *
      * @param modeParameter resource validation mode code to be validated
+     * @param resourceHelper Resource operation provider for loading related Library resources
+     * @param resourceType a valid resource type string
+     * @param operationContext the FHIROperationContext associated with the request
      * @throws FHIROperationException when the resource validation mode code is invalid
      */
-    private void validateModeParameter(Parameter modeParameter) throws FHIROperationException {
+    private void validateModeParameter(ModeType modetype, FHIRResourceHelpers resourceHelper, String resourceType, FHIROperationContext operationContext) throws FHIROperationException {
+        if (modetype != null) {
```

Comment:
this is a style-only comment, but I like to use "guard clauses" for stuff like this.
basically you'd add this here 
```
if (mode == null) {
    return;
}
```
and then unindent everything that follows.
Its just a personal preference thing.

Reply:
This has been fixed as suggested by adding below check and then unindent everything that follows.
 if (modeType == null) {
    return;
}